internal/builders/docker/pkg/builder.go

```diff
@@ -481,25 +481,39 @@ func (c *GitClient) checkoutGitCommit() error {
 func saveToTempFile(verbose bool, readers ...io.Reader) ([]string, error) {
 	var files []string
 	for _, reader := range readers {
-		bytes, err := io.ReadAll(reader)
-		if err != nil {
-			return files, err
-		}
+		var allBytes []byte
 
-		if verbose {
-			if len(bytes) > 0 {
+		for _, reader := range readers {
```

Comment:
Is it intentional to have two nested loops over the same range?

Reply:
unintentional, thx, fixed